src/containerapp/HISTORY.rst

```diff
@@ -8,6 +8,7 @@ upcoming
 * 'az containerapp update': fix bug for mounting secret volumes using --secret-volume-mount
 * 'az containerapp compose create': fixed an issue where the environment's resource group was not resolved from --environment when the input value was a resource id.
 * 'az containerapp replica count', returns the replica count of a container app
+* [Breaking Change] 'az containerapp job create': update default values for container app job properties
```

Comment:
it is suggested to specify the changed params and default values so that they can be more clear for users.

Reply:
```suggestion
[Breaking Change] 'az containerapp job create': add default values for container app job properties --replica-completion-count, --replica-retry-limit, --replica-timeout, --parallelism, --min-executions, --max-executions, --polling-interval
```